de-lib/src/main/java/org/iplantc/de/admin/desktop/client/gin/BelphegorAppsGinModule.java

```diff
@@ -30,17 +30,15 @@
 import org.iplantc.de.admin.desktop.client.toolRequest.view.ToolRequestViewImpl;
 import org.iplantc.de.admin.desktop.client.views.BelphegorView;
 import org.iplantc.de.admin.desktop.client.views.BelphegorViewImpl;
-import org.iplantc.de.admin.desktop.client.workshopAdmin.WorkshopAdminView;
```

Comment:
I was messing with this class. I should have removed inadvertently. But Looks like they are not used anymore.